source/cipher_openssl.c

```diff
@@ -682,16 +682,24 @@ int aws_cryptosdk_sig_verify_start(
     struct aws_allocator *alloc,
     const struct aws_string *pub_key,
     const struct aws_cryptosdk_alg_properties *props) {
+    AWS_PRECONDITION(pctx);
+    AWS_PRECONDITION(alloc);
+    AWS_PRECONDITION(aws_string_is_valid(pub_key));
+    AWS_PRECONDITION(props);
```

Comment:
alpha order

Reply:
its in the same order as the function params.

Reply:
Oh right, perfect then!